zombies/src/main/java/org/phantazm/zombies/map/Round.java

```diff
@@ -101,6 +107,9 @@ public void startRound(long time) {
 
             int prevBestRound = zombiesPlayer.module().getStats().getBestRound();
             zombiesPlayer.module().getStats().setBestRound(Math.max(prevBestRound, roundInfo.round()));
+            zombiesPlayer.getPlayer().ifPresent(player -> {
+                packetHandler.output(player, new RoundStartPacket());
```

Comment:
why have PacketHandler API instead of just using Player#sendPacket

Reply:
automatic serialization